TimeParse.cpp

```diff
@@ -0,0 +1,72 @@
+#include "TimeParse.h"
+#include <iostream>
+
+using namespace std;
+
+TimeParse::TimeParse(string line)
+{
+    doParsing(line);
+}
+void TimeParse::doParsing(string input)
+{
+    int set_count =0, startIndex =0;
+    string word;
+    for(int i=0;i<input.length();i++)
+    {
+        if(input[i] ==':'){
+            word = input.substr(startIndex,i-1);
```

Comment:
How i - 1 ?, this is supposed to be i - startIndex ?

Reply:
done